README.md

```diff
@@ -1 +1,51 @@
-# tdd-project
\ No newline at end of file
+# TDD project
+
+In this project we will implement a class with some methods, should be by doing test-driven development (TDD). The idea is to write tests first and then the code. 
+
+## Built With
+- Ruby
+- rubocop
+
+## Getting Started
+- Make sure you have `Ruby` installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/).
+- Clone this repository by running `git@github.com:Donard97/oop-school-library-app.git` in your command line.
+- Navigate to the repository by running `cd oop-school-library-app`.
+- Run the project by running `ruby main.rb` on the command line.
```

Comment:
- [ ] Let us clean up this section to reflect this project and not the Library project.

Reply:
Thank you for letting us know.